ct-app/aggregator/routes.py

```diff
@@ -153,14 +153,11 @@ async def post_to_db(request: Request):  # pragma: no cover
                 return sanic_text("Table not available", status=500)
 
             log.info(f"Inserting {len(matchs_for_db)} rows into DB")
-
-            for peer, nodes, latencies in matchs_for_db:
-                log.info(f"Inserting {peer} into DB")
-                db.insert(
-                    "raw_data_table",
-                    peer_id=peer,
-                    node_addresses=nodes,
-                    latency_metric=latencies,
+            if len(matchs_for_db) > 0:
+                db.insert_many(
```

Comment:
Make sure to capture potential exceptions raised by this code.

Reply:
Done !